### DISCUSSION TITLE
chyba! pomoc

### DISCUSSION THREAD

**Discussion by johnyss198** [Courseplay/Courseplay_FS22#3609] [General]
![chyba course](https://github.com/user-attachments/assets/4add5a6b-f164-4045-8653-36720741b437)

**Comment by Snigyman** [comment]
And what? This is only inform window :-)

**Comment by johnyss198** [comment]
yes, but it doesn't work for me like it did before the update

---------- Původní e-mail ----------
Od: Snigyman ***@***.***>
Komu: Courseplay/Courseplay_FS22 ***@***.***>
Kopie: johnyss198 ***@***.***>, Author ***@***.***>
Datum: 3. 12. 2024 9:31:20
Předmět: Re: [Courseplay/Courseplay_FS22] chyba! pomoc (Discussion #3609)
"


And what? This is only inform window :-)

—
Reply to this email directly, view it on GitHub
(https://github.com/Courseplay/Courseplay_FS22/discussions/3609#discussioncomment-11445564)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/BNGER5TJ74TY7G6O425VGGT2DVT5NAVCNFSM6AAAAABS5GHCXCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBUGU2TMNA)
.
You are receiving this because you authored the thread. Message ID: <
***@***.***
com>
"

**Reply by Tensuko** [reply]
Then help us by telling what you expect.
All you got to do in that screen is hit ok.
It is just an info notification...
